Makefile

```diff
@@ -31,7 +30,8 @@ all: test
 # The tests are written in Python. Make a virtualenv to handle the dependencies.
 # make doesn't play nicely with custom VENV, intended only for CI usage
 venv: requirements.txt
-	test -d $(VENV) || virtualenv -q --python=$(PYTHON3) $(VENV);\
+	$(PYTHON) -m venv $(VENV);\
```

Comment:
Will it work if python3 is not available? The previous setup I guess it did allow that particular use case

Reply:
no more support for Python2

Reply:
IIUC, the virtualenv installation used to be installed to support python3, that's not anymore the use case, as it does use the python3 binary to install the venv. In other words, I'll keep the previous command:

`test -d $(VENV) || virtualenv -q --python=$(PYTHON3) $(VENV);\`

The above command is backward compatible independently of the python version installed in the system, therefore, it's not necessary to set the PYTHON variable as it will work out of the box.  

I prefer to keep the functionality as it was even though the only required change could be de predefined python3 binary.

WDYT?